src/layouts/main/styles/functions.js

```diff
@@ -0,0 +1,74 @@
+import {css} from 'styled-components';
+import {drawerWidth} from './constants';
+
+/**
+ * To return drawer styles when is open.
```

Comment:
nit: For comments, usually we like to start with a verb. e.g. "Returns drawer styles..."

For here and in other places.

Reply:
Done, also I rewrote almost every comment to try to fit into this recommendation.